lib/fizzy/parser.cpp

```diff
@@ -213,6 +213,8 @@ inline parser_result<Memory> parse(const uint8_t* pos, const uint8_t* end)
 {
     Limits limits;
     std::tie(limits, pos) = parse_limits(pos, end);
+    if (limits.min > 65536 || (limits.max.has_value() && *limits.max > 65536))
```

Comment:
Should I add a constant into `limits.h`?

Reply:
There is already `PageSize`

Reply:
Ah, it's not a page size, then yes, I think it's better to add a new const

Reply:
What should it be named?

Reply:
can be `MemoryPagesValidationLimit`

Reply:
Added.